eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/io/AccessionReportWriter.java

```diff
@@ -119,14 +119,13 @@ public void write(List<? extends AccessionWrapper<ISubmittedVariant, String, Lon
         fileWriter.flush();
     }
 
-    private void writeVariant(Long id, ISubmittedVariant normalizedVariant) throws IOException {
-        ISubmittedVariant variant = denormalizeVariant(normalizedVariant);
-        String vcfLine = variantToVcfLine(id, variant);
+    private void writeVariant(Long id, ISubmittedVariant denormalizedVariant) throws IOException {
+        String vcfLine = variantToVcfLine(id, denormalizedVariant);
         fileWriter.write(vcfLine);
         fileWriter.newLine();
     }
 
-    private ISubmittedVariant denormalizeVariant(ISubmittedVariant normalizedVariant) {
+    protected ISubmittedVariant denormalizeVariant(ISubmittedVariant normalizedVariant) {
```

Comment:
This method is no longer used in this class but in `AccessionWriter`, so I think it is more appropriate to move there this and other methods it depends on, and keep it private.

Reply:
Denormalization and sorting was moved to this class (AccessionReportWriter) istead of having it in the AccessionWriter